runfiles.go

```diff
@@ -89,6 +89,18 @@ func New(opts ...Option) (*Runfiles, error) {
 	}
 	// See section “Runfiles discovery” in
 	// https://docs.google.com/document/d/e/2PACX-1vSDIrFnFvEYhKsCMdGdD40wZRBX3m3aZ5HhVj4CtHPmiXKDCxioTUbYsDydjKtFDAzER5eg7OjJWs3V/pub.
+	// We deviate from the doc (as of 2021-12-30) in one point to work around
```

Comment:
The doc has now been updated, so I'm happy to merge this PR, but please update this comment accordingly first.

Reply:
Done. I dropped everything but the link, which now does explain the process correctly.

Reply:
Thanks!